Makefile

```diff
@@ -1,6 +1,7 @@
 generate-ci:
```

Comment:
should we perhaps create dedicated make targets ? 

`eventing-core`, `ekb`, or the like?